stack.yaml

```diff
@@ -1,13 +1,12 @@
-resolver: lts-15.6
-packages:
-- '.'
+resolver: lts-16.9
+
+ghc-options:
+  $everything: -split-sections
```

Comment:
Looks good to me, could you add a quick comment why the flag is used here as well?  Otherwise I'll probably forget.

Reply:
Sure, I will once I'm home. I should mention that it's not enough just to specify it in a project's `.cabal`, since that won't propagate the setting to its dependencies. Hence its done in the `stack.yaml`, targeting `$everything`.

Reply:
Done.